### ISSUE TITLE
R graphics engine version 15 is not supported by this version of RStudio. 

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/MultiAssayWorkshop#5]
Running this workshop on app.orchestra.cancerdatasci.org (Multi-omic Integration of cBioPortal and TCGA data with MultiAssayExperiment, 2021-07-07) I see the following warning message. It looks like the container needs to be updated.

```
Warning message:
R graphics engine version 15 is not supported by this version of RStudio. The Plots tab will be disabled until a newer version of RStudio is installed.
```

**Comment by lwaldron** [comment]
Attempting a quick fix by changing devel -> RELEASE_3_14 in d421747

**Comment by LiNk-NY** [comment]
Outdated.
